src/Benchmarks/Startup.cs

```diff
@@ -160,6 +160,8 @@ public void Configure(IApplicationBuilder app, ApplicationDbSeeder dbSeeder)
 
             if (Scenarios.Any("Db"))
             {
+                dbContext.Database.EnsureCreated();
```

Comment:
So this is our work around for now to the broken ef migration tool. I'm only doing `EnsureCreated()` here but if we wanted to be extra sure the application boots with the correct Database I could drop the `EnsureDeleted()` prior to this. Given how infrequently the DB models change in this repo I felt that just ensuring its created is sufficient. That way startup time will be snappy :smile:.

Users can always delete the db manually if they want to.